### PR TITLE
Add test class

### PR DESCRIPTION
**Pull request by arnavs**
cc: @mmcky, @jlperla, @nosferican 

OK, I repackaged the `solution` class as `test` and it appears to work as expected.

### CODE REVIEW

File: sphinxcontrib/jupyter/__init__.py [hunk 1]
```diff
@@ -11,6 +11,8 @@ def setup(app):
     app.add_config_value("jupyter_options", None, "jupyter")
     app.add_config_value("jupyter_default_lang", "python3", "jupyter")
     app.add_config_value("jupyter_drop_solutions", False, "jupyter")
+    app.add_config_value("jupyter_drop_tests", False, "jupyter")
```
Comment (mmcky) [inline sphinxcontrib/jupyter/__init__.py]: @arnavs I think the default value should be `True`

Comment (arnavs) [reply sphinxcontrib/jupyter/__init__.py]: OK, no problem. I'll change it.

Comment (mmcky) [reply sphinxcontrib/jupyter/__init__.py]: I should update that for `solutions` also.

**Comment by mmcky** [comment]
thanks @arnavs this looks excellent. I am happy to merge once that default is updated. Thanks also for adding the test case

**Comment by arnavs** [comment]
@mmcky Sounds great. Updated the values for `test`, and did `solution` while I was at it.

**Comment by mmcky** [comment]
thanks @arnavs - merging now.

**Pull request merged**